ui/src/main/js/store/actions/audit.js

```diff
@@ -3,7 +3,7 @@ import * as HTTPErrorUtils from 'common/util/httpErrorUtilities';
 import { unauthorized } from 'store/actions/session';
 import HeaderUtilities from 'common/util/HeaderUtilities';
 
-const FETCH_URL = '/alert/api/audit';
+const FETCH_URL = '/alert/api/audit/failed';
```

Comment:
If this is for a failing request, we should label the variable to be more specific.  Something like FETCH_URL_FAIL or FETCH_URL_ERROR

Reply:
Updated